### PR TITLE
update subscriptionUrl for Iranian filter

### PR DESCRIPTION
**Pull request by slavaleleka**
https://uploads.adguard.com/slbdzdrc03en.png

### CODE REVIEW

File: filters/ThirdParty/filter_235_adblock-iran/metadata.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "expires": "4 days",
   "displayNumber": 2,
   "groupId": 7,
-  "subscriptionUrl": "https://gitcdn.xyz/repo/farrokhi/adblock-iran/master/filter.txt",
+  "subscriptionUrl": "https://raw.githubusercontent.com/farrokhi/adblock-iran/master/filter.txt",
```
Comment (Alex-302) [inline filters/ThirdParty/filter_235_adblock-iran/metadata.json:10]: template.txt must be changed

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: filters/ThirdParty/filter_235_adblock-iran/metadata.json (hunk 1 above)
Comment (Alex-302) [reply filters/ThirdParty/filter_235_adblock-iran/metadata.json:10]: Also what about changing `trustLevel`? It seems it has not bad scriptlets.

**Review by ameshkov — APPROVED** [review]

File: filters/ThirdParty/filter_235_adblock-iran/metadata.json (hunk 1 above)
Comment (ameshkov) [reply filters/ThirdParty/filter_235_adblock-iran/metadata.json:10]: @Alex-302 let's open a separate PR for that

Comment (slavaleleka) [reply filters/ThirdParty/filter_235_adblock-iran/metadata.json:10]: it really has not so many scriptlet rules and they seems to be clear

**Pull request merged**